docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst

```diff
@@ -532,6 +532,28 @@ Deriving wasting transition probabilities
 Wasting model
 ^^^^^^^^^^^^^
 
+.. important::
+
+  We will model wasting transitions **only** among simulants at least six months of age. Simulants should be initialized into a wasting model state at birth with a birth prevalence equal to the wasting risk exposure among the 6-11 month age group (age_group_id=389, or postneonatal age_group_id=4 if using GBD 2019 instead of GBD 2020).
+
+  All wasting transition rates should equal zero among all ages under 6 months.
+
+  Wasting transition rates should be informed by the data tables below for ages over 6 months.
+
+  Related assumptions/limitations: 
+
+  #. We will apply the wasting exposure distribution among the 6-11 month age group to the 1-5 month age group. Because wasting risk effects are present among the 1-5 month age group, this may cause lack of validation to the outcomes affected by wasting for this age group, including measles/diarrheal diseases/PEM DALYs. 
+
+  #. When the birthweight and wasting risk exposure at birth correlation is implemented, it will cause simulants with a greater neonatal mortality (due to brithweight exposure) to be initialized into more severe wasting states. This will cause the wasting exposure distribution to shift to less severe wasting states over the neonatal period as simulants with lower birthweights (and more severe wasting states due to the birthweight and wasting exposure correlation) die. 
+
+  The magnitude of the bias introduced by this modeling strategy due to the two points above should be investigated upon implementation to determine if different modeling strategies are necessary. This should be done by comparing the wasting exposure and wasting-affected outcomes in the simulation output to the GBD inputs by age group.
+
+  NOTE:
+
+  - The modeling decision not to model wasting transitions among simulants less than six months of age is due to the reliance of the wasting model transition rates on the wasting treatment model and the lack of data to inform treatment-related transition rates among this age group. Note that a sensitivity analysis scenario that includes infants less than six months of age in the treatment model may be performed in the future.
+
+  - The decision to initialize simulants with the birth prevalence of the 6-11 month age group with all wasting transition rates equal to zero before six months of age rather than have a static wasting exposure model among simulants less than six months of age (similar to the stunting exposure model) was chosen due to the ease of engineering implementation and will be revisited if it introduces a large bias.
```

Comment:
Shouldn't this be the prevalence in the 1-6 months age group? Since that's the prevalence we'd expect to be reflected in a vivarium sim in which we were modelling wasting in that age group.

Reply:
@rmudambi Hmm, I chose the 6-11 month age group because that is the age group where we will be measuring our intervention impact, so we want to prioritize getting exposure correct in this age group. The exposures between these groups are very similar, so I don't think that it will cause an especially large bias in the mortality rate among 1-5 month olds. Notably, exposure is slightly greater among the 6-11 month age group than the 1-5 month age group. Given that we know our strategy will bias exposure lower (upon the BW/wasting correlation implementation), selecting the higher exposure group will offset this bias more than selecting the lower exposure group.

Does that address your concern?